democluster: expose flag for workload rate limit on demo

otan opened the pull request:
Release note (cli change): The 25 max QPS rate limit for workloads on
`cockroach` demo can now be configured with a `workload-max-qps` flag.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/70642)
<!-- Reviewable:end -->

knz reviewed (approved):
> Reviewed 5 of 5 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/70642)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @otan)

___
*[pkg/cli/democluster/demo_cluster.go, line 70 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/70642#-MkLu9MX-9ZOjSR8vn7b:-MkLu9MY4ydzynaTGVhD:b-pi9d9n) ([raw file](https://github.com/cockroachdb/cockroach/blob/6ee0079c13fa85ea26a304ea6bf0a43dcd077901/pkg/cli/democluster/demo_cluster.go#L70)):*
> ```Go
> 	sqlFirstPort  int
> 
> 	workloadMaxQPS int
> ```

remove this


<!-- Sent from Reviewable.io -->

otan reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/70642)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @knz)

___
*[pkg/cli/democluster/demo_cluster.go, line 70 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/70642#-MkLu9MX-9ZOjSR8vn7b:-MkM-ZMw-OiYVT6cL7Qr:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/6ee0079c13fa85ea26a304ea6bf0a43dcd077901/pkg/cli/democluster/demo_cluster.go#L70)):*
<details><summary><i>Previously, knz (kena) wrote…</i></summary><blockquote>

remove this
</blockquote></details>

Done.


<!-- Sent from Reviewable.io -->

otan commented:
thanks! 

bors r=knz

otan committed:
democluster: expose flag for workload rate limit on demo

Release note (cli change): The 25 max QPS rate limit for workloads on
`cockroach` demo can now be configured with a `workload-max-qps` flag.

otan commented:
oops!

bors r=knz
